RealmSwift/Sync.swift

```diff
@@ -218,7 +218,7 @@ public enum ClientResetMode {
     case manual
     /// - see: `RLMClientResetModeDiscardLocal` for more details on `.discardLocal` behavior
     ///
-    /// The first `.discardLocal` function argument notifies prior to a client reset occurring.
+    /// The first `.discardLocal` function argument, `((Realm) -> Void)? = nil`, is executed prior to a client reset occurring.
```

Comment:
I don't think it's useful to repeat the type of the argument in the docs, as that's already displayed.

Reply:
Okay will take it out. I was on the fence since in swift these blocks don't have a proper name.